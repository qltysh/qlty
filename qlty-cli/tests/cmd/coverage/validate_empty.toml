args = [
  "coverage",
  "validate",
  "coverage.zip",
  "--json"
]
bin.name = "qlty"
status.code = 4

[env]
remove = ["GITHUB_ACTIONS"]
