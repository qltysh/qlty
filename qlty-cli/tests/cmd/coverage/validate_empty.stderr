❌ Coverage validation failed
No coverage data found
